pkgs/by-name/db/dbt-duckdb/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+  python3Packages.buildPythonApplication rec {
+    pname = "dbt-duckdb";
+    version = "1.9.6";
+    pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "duckdb";
+    repo = "dbt-duckdb";
+    tag = version;
+    hash = "sha256-zuSOM5Vfr89aVGDcVQtK1Y4c3hRcL8pRscnvpeP6jbU=";
+  };
+
+  build-system = [
+    python3Packages.pkgs.setuptools
+    python3Packages.pkgs.wheel
```

```suggestion
    python3Packages.setuptools
    python3Packages.wheel
```
please, check locally or at least the CI logs to see if smth doesn't work